src/plugins/data_views/public/saved_objects_client_wrapper.ts

```diff
@@ -17,42 +17,55 @@ import {
 
 type SOClient = Pick<SavedObjectsClient, 'find' | 'resolve' | 'update' | 'create' | 'delete'>;
 
-const simpleSavedObjectToSavedObject = <T>(simpleSavedObject: SimpleSavedObject): SavedObject<T> =>
+const simpleSavedObjectToSavedObject = <T extends SavedObjectAttributes>(
+  simpleSavedObject: SimpleSavedObject
+): SavedObject<T> =>
   ({
     version: simpleSavedObject._version,
     ...omit(simpleSavedObject, '_version'),
   } as any);
 
 export class SavedObjectsClientPublicToCommon implements SavedObjectsClientCommon {
   private savedObjectClient: SOClient;
+
   constructor(savedObjectClient: SOClient) {
     this.savedObjectClient = savedObjectClient;
   }
-  async find<T = unknown>(options: SavedObjectsClientCommonFindArgs) {
+
+  async find<T extends SavedObjectAttributes = SavedObjectAttributes>(
+    options: SavedObjectsClientCommonFindArgs
+  ) {
     const response = (await this.savedObjectClient.find<T>(options)).savedObjects;
     return response.map<SavedObject<T>>(simpleSavedObjectToSavedObject);
   }
 
-  async get<T = unknown>(type: string, id: string) {
+  async get<T extends SavedObjectAttributes = SavedObjectAttributes>(type: string, id: string) {
```

Comment:
This is the second thing a bit bothersome with that `T extends SavedObjectAttributes = SavedObjectAttributes` type change. We can no longer use `unknown` as a default for higher-level APIs when the shape of the attributes is not relevant, forcing to adapt all these higher-level APIs to also define the correct generic types.